cime_config/user_nl_blom_outvol_reduced

```diff
@@ -0,0 +1,534 @@
+  SRF_SILICA        = 2, 2
+  SRF_DIC           = 2, 2
+  SRF_PHYTO         = 2, 2
+  SRF_EXPORT        = 2, 2
+  SRF_EXPOSI        = 2, 2
+  SRF_EXPOCA        = 2, 2
+  SRF_KWCO2         = 2, 2
+  SRF_KWCO2SOL      = 2, 2
+  SRF_FCO2          = 2, 2
+  SRF_PCO2          = 2, 2
+  SRF_XCO2          = 2, 2
+  SRF_PCO2_GEX      = 2, 2
+  SRF_PN2OM         = 2, 2
+  SRF_PNH3          = 2, 2
+  SRF_PH            = 2, 2
+  SRF_CO2SOL        = 2, 2
+  SRF_CO2FXD        = 2, 2
+  SRF_CO2FXU        = 2, 2
+  SRF_OXFLUX        = 2, 2
+  SRF_NIFLUX        = 2, 2
+  SRF_N2OFX         = 2, 2
+  SRF_DMSFLUX       = 2, 2
+  SRF_DMS           = 2, 2
+  SRF_DMSPROD       = 2, 2
+  SRF_DMS_BAC       = 2, 2
+  SRF_DMS_UV        = 2, 2
+  SRF_ATMBROMO      = 2, 2
+  SRF_ATMCO2        = 2, 2
+  SRF_ATMO2         = 2, 2
+  SRF_ATMN2         = 2, 2
+  SRF_ATMNH3        = 2, 2
+  SRF_ATMN2O        = 2, 2
+  SRF_NATDIC        = 2, 2
+  SRF_NATALKALI     = 2, 2
+  SRF_NATPCO2       = 2, 2
+  SRF_NATCO2FX      = 2, 2
+  SRF_NATPH         = 2, 2
+  SRF_CO213FXD      = 2, 2
+  SRF_CO213FXU      = 2, 2
+  SRF_CO214FXD      = 2, 2
+  SRF_CO214FXU      = 2, 2
+  SRF_CFC11         = 2, 2
+  SRF_CFC12         = 2, 2
+  SRF_SF6           = 2, 2
+  SRF_BROMO         = 2, 2
+  SRF_BROMOFX       = 2, 2
+  INT_PHOSY         = 2, 2
+  INT_NFIX          = 2, 2
+  INT_DNIT          = 2, 2
+  INT_BROMOPRO      = 2, 2
+  INT_BROMOUV       = 2, 2
+  FLX_CAR0100       = 2, 2
+  FLX_CAR0500       = 2, 2
+  FLX_CAR1000       = 2, 2
+  FLX_CAR2000       = 2, 2
+  FLX_CAR4000       = 2, 2
+  FLX_CAR_BOT       = 2, 2
+  FLX_BSI0100       = 2, 2
+  FLX_BSI0500       = 2, 2
+  FLX_BSI1000       = 2, 2
+  FLX_BSI2000       = 2, 2
+  FLX_BSI4000       = 2, 2
+  FLX_BSI_BOT       = 2, 2
+  FLX_CAL0100       = 2, 2
+  FLX_CAL0500       = 2, 2
+  FLX_CAL1000       = 2, 2
+  FLX_CAL2000       = 2, 2
+  FLX_CAL4000       = 2, 2
+  FLX_CAL_BOT       = 2, 2
+  FLX_DUST0100      = 2, 2
+  FLX_DUST0500      = 2, 2
+  FLX_DUST1000      = 2, 2
+  FLX_DUST2000      = 2, 2
+  FLX_DUST4000      = 2, 2
+  FLX_DUST_BOT      = 2, 2
+  FLX_NDEPNHX       = 2, 2
+  FLX_NDEPNOY       = 2, 2
+  FLX_OALK          = 0, 0
+  FLX_SEDIFFAL      = 0, 2
+  FLX_SEDIFFIC      = 0, 2
+  FLX_SEDIFFN2      = 0, 2
+  FLX_SEDIFFN2O     = 0, 2
+  FLX_SEDIFFNH4     = 0, 2
+  FLX_SEDIFFNO2     = 0, 2
+  FLX_SEDIFFNO3     = 0, 2
+  FLX_SEDIFFOX      = 0, 2
+  FLX_SEDIFFPH      = 0, 2
+  FLX_SEDIFFSI      = 0, 2
+  FLX_BURSSO12      = 0, 2
+  FLX_BURSSSC12     = 0, 2
+  FLX_BURSSSSIL     = 0, 2
+  FLX_BURSSSTER     = 0, 2
+
+! All layer-output disabled for reduced outpu setting
+
+  LYR_AGG_AVDC      = 0, 0
+  LYR_AGG_AVDP      = 0, 0
+  LYR_AGG_AVRHOP    = 0, 0
+  LYR_AGG_B         = 0, 0
+  LYR_AGG_DF        = 0, 0
+  LYR_AGG_DMAX      = 0, 0
+  LYR_AGG_STICK     = 0, 0
+  LYR_AGG_STICKF    = 0, 0
+  LYR_AGG_VPOR      = 0, 0
+  LYR_AGG_VRHOF     = 0, 0
+  LYR_AGG_WS        = 0, 0
+  LYR_ANH4          = 0, 0
+  LYR_ANMX_N2_PROD  = 0, 0
```

Comment:
Are we going to use this user_nml also for continued tuning purposes? If so, some layer information (annual) for the N-cycle would be appreciated - if, yes, I can provide a list.

Reply:
What do you mean by "continued tuning purposes"?

You would want layer output rather than level?